db/seeds.rb

```diff
@@ -8,11 +8,17 @@
 if Figaro.env.enable_sso_integration == 'false'
   user = User.create(email: 'superadmin@barito.com', username: 'superadmin', password: '123456', password_confirmation: '123456')
 end
-GroupUser.create(user: user, group: superadmin_group, role: AppGroupRole.find_by_name('admin'))
-GroupUser.create(user: user, group: global_viewer_group, role: AppGroupRole.find_by_name('admin'))
+GroupUser.create(user: user, group: superadmin_group, role: AppGroupRole.find_by_name('admin'), expiration_date: Time.now + 10.days)
+GroupUser.create(user: user, group: global_viewer_group, role: AppGroupRole.find_by_name('admin'), expiration_date: Time.now + 10.days)
```

Comment:
after we have this, it will grant them access for the next 10 days only? should we keep this as nil? so it won't affect this feature.

Reply:
Updated